typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow/decorators/itemDecoratorStrategy.ts

```diff
@@ -57,7 +64,30 @@ export default class ItemDecoratorStrategy implements CanvasItemDecoratorStrateg
 		return this.sourceFilePrinter.printSourceFile(methodNode, sourceFile);
 	}
 
+	/**
+	 * Prints out the item
+	 *
+	 * - `out-name` is the target canvas item to be called after the item is executed
+	 */
 	printItem(pos: number): string {
-		throw new Error("Method not implemented.");
+		const stringBuilder = new StringBuilderClass("", "");
+		const targetItem = findTargetItem(this.itemInfo.target, pos, this.itemInfo);
+
+		stringBuilder.append(`<workflow-item`
+			+ ` name="item${pos}"`
+			+ ` out-name="${targetItem}"`
+			+ ` type="${this.getCanvasType()}"`
+			+ ">").appendLine();
+		stringBuilder.indent();
+		stringBuilder.append(`<script encoded="false"><![CDATA[${this.itemInfo.sourceText}]]></script>`).appendLine();
+
+		stringBuilder.append(`<display-name><![CDATA[${this.itemInfo.name}]]></display-name>`).appendLine();
+		stringBuilder.appendContent(buildItemParameterBindings(this, InputOutputBindings.IN_BINDINGS));
+		stringBuilder.appendContent(buildItemParameterBindings(this, InputOutputBindings.OUT_BINDINGS));
+		stringBuilder.append(`<position x="${225 + 160 * (pos - 1)}.0" y="55.40909090909091" />`).appendLine();
```

Comment:
It is a good idea to externalize the above numbers to a constant.

Reply:
Take a look in the other comment